pages/common/magento.md

```diff
@@ -0,0 +1,35 @@
+# magento
+
+> A CLI for managing the Magento PHP framework.
+
+- Update the database after enabling modules:
```

Comment:
Should this command go after the enable/disable of modules ?

Reply:
Good point :+1: